### ISSUE TITLE
Find a better way for selecting scenes

### ISSUE THREAD

**Issue by jimver04** [Envisage-H2020/Wind-Energy-Simulation#12]
Pano, 
instead of a menu with tiles, can you foresee another way of selecting scenes ?
According to the teachers, they wanted a fly-over way that centraly views all the scenes in 3D way.
It is like RTS game where you select your level by previewing the levels

**Comment by PanMig** [comment]
Yes, we will create a 3d scene, with a camera that will provide a look from above. Then the player will be able to hover over different areas of the map and get a list of characteristics and future costs considering the installation of the farm. Then he would be able to click on the desired area and he will be transported to the chosen area of the map.

Full game design document will be released in the upcoming days that will explain these actions.
